Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -320,7 +320,51 @@ The dynamic analysis of this section comprises of two parts: the actual license
+
+###### objection
+
+objection contains bundles module to work with iOS Bundles. This module has two commands `list_bundles` and `list_frameworks`.
+
+`list_bundles` command lists all of the application’s non framework bundles. The output of both commands contains executable name, bundle id, version of the library and path to the library.
+
+```bash
+...itudehacks.DVIAswiftv2.develop on (iPhone: 13.2.3) [usb] # ios bundles list_bundles
+Executable    Bundle                                       Version  Path
+------------  -----------------------------------------  ---------  -------------------------------------------
+DVIA-v2       com.highaltitudehacks.DVIAswiftv2.develop          2  ...-1F0C-4DB1-8C39-04ACBFFEE7C8/DVIA-v2.app
+CoreGlyphs    com.apple.CoreGlyphs                               1  ...m/Library/CoreServices/CoreGlyphs.bundle
+```
+
+`list_frameworks` command lists all of the application’s bundles that represent frameworks.
+
+```bash
+...itudehacks.DVIAswiftv2.develop on (iPhone: 13.2.3) [usb] # ios bundles list_frameworks
+Executable      Bundle                                     Version    Path
+--------------  -----------------------------------------  ---------  -------------------------------------------
+Bolts           org.cocoapods.Bolts                        1.9.0      ...8/DVIA-v2.app/Frameworks/Bolts.framework
+Parse           org.cocoapods.Parse                        1.17.3     ...8/DVIA-v2.app/Frameworks/Parse.framework
+                                                                      ...rivateFrameworks/ASEProcessing.framework
+                                                                      ...teFrameworks/AppleNeuralEngine.framework
+DVIA-v2         com.highaltitudehacks.DVIAswiftv2.develop  2.0        ...-1F0C-4DB1-8C39-04ACBFFEE7C8/DVIA-v2.app
+                                                                      ...rameworks/CoreLocationProtobuf.framework
+Flurry_iOS_SDK  org.cocoapods.Flurry-iOS-SDK               10.0.2     ....app/Frameworks/Flurry_iOS_SDK.framework
+RealmSwift      org.cocoapods.RealmSwift                   4.1.1      ...A-v2.app/Frameworks/RealmSwift.framework
+Realm           org.cocoapods.Realm                        4.1.1      ...8/DVIA-v2.app/Frameworks/Realm.framework
+                                                                      ...vateFrameworks/CoreDuetContext.framework
+                                                                      ...ystem/Library/Frameworks/IOKit.framework
+```
+
+###### MobSF
+
+After obtaining the application ipa file, it can be uploaded to the MobSF. MobSF lists external and system libraries, however it provides wrong versions of some libraries. This can be seen by comparing objection results with mobsf results.
```

Comment:
> however it provides wrong versions of some libraries

I would slightly rephrase this, and possibly place it as its own sentence. "However, it occassionally provides the incorrect version of some libraries."

> results with mobsf results

Perhaps capitalize this consistently: MobSF.

Reply:
Corrected, thanks for the comments :)